### PR TITLE
Fix: Add tests to cover default magic number options (fixes #5385)

### LINKED ISSUE #5385: Add tests to cover default options for complexity, max-depth, max-nested-callbacks, max-parms, max-statements
See discussion here https://github.com/eslint/eslint/pull/5349

### PR DESCRIPTION
**Pull request by bsbeeks**

### CODE REVIEW

File: tests/lib/rules/complexity.js [hunk 1]
```diff
@@ -16,6 +16,23 @@ var rule = require("../../../lib/rules/complexity"),
 // Helpers
 //------------------------------------------------------------------------------
 
+/**
+ * Generates a code string with the amount of complexity specified in the parameter
+ * @param {int} complexity The level of complexity
+ * @returns {string} Code with the amount of complexity specified in the parameter
+ * @private
+ */
+function creatComplexity(complexity) {
```
Comment (btmills) [inline tests/lib/rules/complexity.js]: Ha! Nice.

**Comment by btmills** [comment]
`max-nested-callbacks` [already has](https://github.com/eslint/eslint/blob/master/tests/lib/rules/max-nested-callbacks.js#L70-L73) an invalid test for the default options, so no changes needed there.

LGTM, just waiting for CI to pass.

File: tests/lib/rules/complexity.js (hunk 1 above)
Comment (jfmengels) [inline tests/lib/rules/complexity.js]: creatComplexity --> createComplexity (sorry about the nitpicking)

Comment (bsbeeks) [inline tests/lib/rules/complexity.js]: :+1:

**Comment by bsbeeks** [comment]
@btmills fixed a spelling error. Once CI passes it should be good to go.

**Comment by btmills** [comment]
Nice catch @jfmengels, and thanks @bsbeeks! Looks good!

**Pull request merged**